web/src/beta/components/InsertionButton/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from "react";
+
+import { styled, useTheme } from "@reearth/services/theme";
+
+import Icon from "../Icon";
+
+type Props = {
+  onClick?: () => void;
+};
+
+const InsertionButton: FC<Props> = ({ onClick }) => {
+  const theme = useTheme();
+
+  return (
+    <Box onClick={onClick}>
+      <Icon
+        icon={"plus"}
+        style={{
+          width: "9.75px",
+          height: "9.75px",
+          background: theme.general.select,
+          borderRadius: "50%",
+          padding: "2px",
+          color: theme.general.bg.transparent,
+        }}
```

Comment:
Please 99% of the time make use of styled components for styles.
In this case just do
```
const StyledIcon = styled(Icon)`
          width: 9.75px;
          height: 9.75px;
          background: ${({theme}) => theme.general.select};
          borderRadius: 50%;
          padding: 2px;
          color: ${({theme}) => theme.general.bg.transparent};
`;
```